### DISCUSSION TITLE
Pressing a key while another is already held down causes duplicate

### DISCUSSION THREAD

**Discussion by moe1011** [JohannesKlauss/react-hotkeys-hook#995] [Q&A]
Hi,
So i'm building a calculator app for a project but i'm running into a weird issue that i'm not sure if it is a bug or what.
As an example, if i click 2 on my keyboard, but 1 was previously held down the the input will be 1 2, instead of just 2.
Not sure if this is a bug, or if there is a way around this if anyone knows.

**Comment by JohannesKlauss** [comment]
Please provide a reproducible with the expected behavior and the actual behavior explained. There is a bug template in the issues section that guides you through the process of creating a bug ticket
